src/lib/modules/charts/services/image-export.service.ts

```diff
@@ -0,0 +1,39 @@
+import { Injectable } from '@angular/core';
+import * as SaveSvg from 'save-svg-as-png';
+
+/*
+ * Generates image of D3 chart SVG for download
+ */
+@Injectable()
+export class ImageExportService {
+
+    /**
+     * Options to pass when converting SVG to PNG
+     *
+     * @param parentSelector - Name of parent of chart, used to rewrite CSS selectors
+     */
+    private chartOptions(parentSelector: string) {
```

Comment:
This used to be an object; apologies for including the switch to a function in the same commit that brings it over into the repo. There was a lot of friction to getting the Components repo linked in to both Temperate and the Lab, and by the time I got everything working, the WiP commits weren't in a place where it was super easy to split that functionality change apart. I'm happy to dive in and break this apart if you want to keep the commit history clean.